test/test_integration_credentials_keyring.py

```diff
@@ -315,9 +317,9 @@ def test_simple_login(self):
         utils.setup_cli(self)
```

Comment:
the title isn't representative of the test. test_cli_simple_login_logout_flow

I also think this isn't run in netrc flow with the mock accessible keyring = False